dependabot_time_to_merge.rb

```diff
@@ -24,33 +24,26 @@ def govuk_repos
 end
 
 def dependabot_prs
-  @count = 0
   client.search_issues("is:pr user:alphagov is:closed author:app/dependabot archived:false", per_page: 100)
   
   last_response = client.last_response
 
   return [] if last_response.data.items.empty?
 
   pulls = []
-  pulls << last_response.data.items
-  puts "Fetching first page of results... (API call ##{@count += 1})"
 
-  until last_response.rels[:next].nil?
-    sleep 60 if (current_page(last_response) % 3).zero?
-    last_response = last_response.rels[:next].get
+  10.times do |i| # GitHub API limit
```

Comment:
I've found some evidence to back this up!

```suggestion
  # GitHub Search API has a limit of 1000 results,
  # so we should only loop through 100-results-per-page 10 times.
  # https://docs.github.com/en/rest/search#about-the-search-api
  10.times do |i|
```